content/en/account_management/audit_trail_events.md

```diff
@@ -0,0 +1,182 @@
+| API key (Org settings)         | An API key is accessed, listed, created, or deleted in the Organization Settings page.        | `@evt.name:Authentication @asset.type:api_key`          |
+| Application key (Org settings) | An application key is accessed, listed, created, or deleted in the Organization Settings page.| `@evt.name:Authentication @asset.type:application_key`  |
+| Public API key (Org settings)  | A public API key is accessed, listed, created, or deleted in the Organization Settings page.  | `@evt.name:Authentication @asset.type:public_api_key`   |
+| User login                     | A user logs into Datadog and the authentication method used.                                  | `@evt.name:Authentication @action:login`                |
+
+### Cloud Security Platform events
+| Name | Description of audit event                                          | Query in audit explorer                           |
+| ---- | ------------------------------------------------------------------- | --------------------------------------------------|
+| CWS agent rule | A user accessed (fetched) a CWS agent rule in the Cloud Security Platform.| `@evt.name:“Cloud Security Platform” @asset.type:cws_agent_rule @action:accessed` |
+| Notification profile | A user created, updated, or deleted a notification profile in the Cloud Security Platform. | `@evt.name:"Cloud Security Platform" @asset.type:notification_profile` |
+| Security rule | A user updated, deleted, or created a security rule and the previous and new values for the rule. | `@evt.name:"Cloud Security Platform" @asset.type:security_rule` |
+| Security signal | A user modified the state of a signal or assigned the signal to a user, and the previous and new values for the signal. | `@evt.name:"Cloud Security Platform" @asset.type:security_signal @action:modified` |
+
+### Dashboard events
+
+| Name               | Description of audit event                                                                        | Query in audit explorer                                               |
+| -------------------| ------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------   |
+| Dashboard created | A dashboard is created and the new JSON value for the dashboard.                                    | `@evt.name:Dashboard @asset.type:dashboard @action:created`             |
+| Dashboard deleted | A dashboard is deleted and the previous JSON value for the dashboard.                              | `@evt.name:Dashboard @asset.type:dashboard @action:deleted`             |
+| Dashboard embedded (Roadie) | A Datadog dashboard is [embedded into a third party][2] and a user views the dashboard.                      | `@evt.name:Dashboard @asset.type:embed @action:accessed`                |
```

Comment:
It's not clear what Roadie means in this context. And I don't understand how the link helps - it just goes to the Audit Trail docs.

Reply:
Oops, fixed the link.